notebooks/03_evaluate/comparison.ipynb

```diff
@@ -373,16 +466,16 @@
        "      <td>100k</td>\n",
        "      <td>als</td>\n",
        "      <td>10</td>\n",
-       "      <td>0.005850</td>\n",
-       "      <td>0.049102</td>\n",
-       "      <td>0.050849</td>\n",
-       "      <td>0.018680</td>\n",
-       "      <td>0.948764</td>\n",
-       "      <td>0.742875</td>\n",
-       "      <td>0.283734</td>\n",
-       "      <td>0.288698</td>\n",
-       "      <td>3.484409</td>\n",
-       "      <td>10.361868</td>\n",
+       "      <td>0.005232</td>\n",
```

Comment:
@gramhagen would it make sense to include results from a VW algo here?

Reply:
+1 to that

Reply:
is the current plan to update the comparison notebook so we have apples to apples for all algos?
if you need current results I can provide them